src/main/java/org/prebid/server/bidder/sharethrough/SharethroughMarkupUtil.java

```diff
@@ -0,0 +1,64 @@
+package org.prebid.server.bidder.sharethrough;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.vertx.core.json.Json;
+import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.prebid.server.bidder.sharethrough.model.StrUriParameters;
+import org.prebid.server.bidder.sharethrough.model.bidResponse.ExtImpSharethroughResponse;
+
+import java.util.Base64;
+
+class SharethroughMarkupUtil {
+
+    private SharethroughMarkupUtil() {
+    }
+
+    static String getAdMarkup(ExtImpSharethroughResponse strResponse, StrUriParameters strUriParameters) {
+        final StringBuilder tmplBody = new StringBuilder();
+
+        final String strRespId = String.format("str_response_%s", strResponse.getBidId());
+        final String arId = strResponse.getAdserverRequestId();
+        final String jsonStrResponse = parseResponseToString(strResponse);
+        if (StringUtils.isBlank(jsonStrResponse)) {
+            return "";
+        }
+
+        final String encodedJsonResponse = Base64.getEncoder().encodeToString(jsonStrResponse.getBytes());
+
+        tmplBody.append("<img src=\"//b.sharethrough.com/butler?type=s2s-win&arid=").append(arId).append("\" />\n")
+                .append("\t\t<div data-str-native-key=\"").append(strUriParameters.getPkey()).append("\" ")
+                .append("data-stx-response-name=\"").append(strRespId).append("\"></div>\n")
+                .append("\t\t<script>var ").append(strRespId).append(" = ")
+                .append("\"").append(encodedJsonResponse).append("\"</script>\n");
+
+        if (BooleanUtils.toBoolean(strUriParameters.getIframe())) {
+            tmplBody.append("\n\t\t<script src=\"//native.sharethrough.com/assets/sfp.js\"></script>\n");
+        } else {
+            tmplBody.append("\t\t\t<script src=\"//native.sharethrough.com/assets/sfp-set-targeting.js\"></script>\n")
```

Comment:
I wonder if we could make this as a single String since it is basically a static value, divided for visual convenience. I got no problem with this, but lets see what @rpanchyk thinks.